app/src/releaseLocal/res/values/environment_config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="feeRelayerBaseUrl">https://fee-relayer.key.app/</string>
+
+    <string name="torusNetwork">cyan</string>
```

Comment:
will the releaseStore and releaseFirebase use this strings.xml too?

Reply:
Yes, it should